apps/test-suite/tests/Calendar.js

```diff
@@ -318,7 +327,7 @@ export async function test(t) {
       t.it('creates an event with the recurrence rule', async () => {
         const eventId = await createTestEventAsync(calendarId, {
           recurrenceRule: {
-            endDate: new Date(2019, 3, 5),
+            endDate: +new Date(2019, 3, 5),
```

Comment:
```suggestion
            endDate: new Date(2019, 3, 5).getTime(),
```
I think explicitly calling getTime is a bit clearer to me.